diff -up qmc2/inst/qmc2.ini.template.fedora qmc2/inst/qmc2.ini.template
--- qmc2/inst/qmc2.ini.template.fedora	2014-03-13 18:05:21.010845945 +0100
+++ qmc2/inst/qmc2.ini.template	2015-07-14 21:20:45.196473431 +0200
@@ -2,8 +2,29 @@
 qmc2-sdlmame\FilesAndDirectories\DataDirectory=DATADIR/qmc2/
 qmc2-sdlmess\FilesAndDirectories\DataDirectory=DATADIR/qmc2/
 qmc2-sdlume\FilesAndDirectories\DataDirectory=DATADIR/qmc2/
+FilesAndDirectories\MameHistoryDat=DATADIR/mame/history.dat
+FilesAndDirectories\MessSysinfoDat=DATADIR/mame/history.dat
+FilesAndDirectories\MameInfoDat=DATADIR/mame/mameinfo.dat
+FilesAndDirectories\MessInfoDat=DATADIR/mame/messinfo.dat
 
 [MAME]
+Configuration\Global\rompath="DATADIR/mame/roms;DATADIR/mame/chds"
+Configuration\Global\hashpath=DATADIR/mame/hash
+Configuration\Global\samplepath=DATADIR/mame/samples
+Configuration\Global\artpath="DATADIR/mame/artwork;DATADIR/mame/effects"
+Configuration\Global\ctrlrpath=DATADIR/mame/ctrlr
+Configuration\Global\fontpath=DATADIR/mame/fonts
+Configuration\Global\cheatpath=DATADIR/mame/cheat
+Configuration\Global\crosshairpath=DATADIR/mame/crosshair
+Configuration\Global\inipath="$HOME/.mame/ini;/etc/mame"
+Configuration\Global\cfg_directory=$HOME/.mame/cfg
+Configuration\Global\nvram_directory=$HOME/.mame/nvram
+Configuration\Global\input_directory=$HOME/.mame/inp
+Configuration\Global\state_directory=$HOME/.mame/sta
+Configuration\Global\snapshot_directory=$HOME/.mame/snap
+Configuration\Global\diff_directory=$HOME/.mame/diff
+Configuration\Global\comment_directory=$HOME/.mame/comments
+Configuration\Global\video=opengl
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMAME/template.xml
 FilesAndDirectories\PreviewDirectory=DATADIR/qmc2/prv/
 FilesAndDirectories\FlyerDirectory=DATADIR/qmc2/fly/
@@ -18,6 +39,10 @@ FilesAndDirectories\SoftwareNotesFolder=
 FilesAndDirectories\SoftwareNotesTemplate=DATADIR/qmc2/swn/template.html
 FilesAndDirectories\SystemNotesFolder=DATADIR/qmc2/gmn/
 FilesAndDirectories\SystemNotesTemplate=DATADIR/qmc2/gmn/template.html
+FilesAndDirectories\ExecutableFile=/usr/bin/mame
+FilesAndDirectories\SoftwareInfoDB=DATADIR/mame/history.dat
+FilesAndDirectories\CatverIni=DATADIR/mame/Catver.ini
+FilesAndDirectories\CategoryIni=DATADIR/qmc2/cat/category.ini
 
 [MESS]
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMESS/template.xml
