diff -up qmc2-0.43/sdlmame/scripts/os-detect.sh.os-detect qmc2-0.43/sdlmame/scripts/os-detect.sh
--- qmc2-0.43/sdlmame/scripts/os-detect.sh.os-detect	2014-03-13 18:04:23.612684676 +0100
+++ qmc2-0.43/sdlmame/scripts/os-detect.sh	2014-04-26 19:28:20.221897625 +0200
@@ -7,15 +7,20 @@ DIST="`uname -r`"
 
 if [ "${OS}" = "Linux" ] ; then
   if [ -f /etc/os-release ] ; then
-    DIST="`cat /etc/os-release | tr '\n' ' ' | sed 's/\ VERSION_ID.*//' | sed 's/\ (.*)//' | sed 's/NAME=//g' | sed 's/VERSION=//g' | sed 's/\"//g'`"
+    VERSION=$(cat /etc/os-release | egrep "\bVERSION_ID\b" | sed 's/VERSION_ID=//g' | sed 's/\"//g')
+    NAME=$(cat /etc/os-release | egrep "\bNAME\b" | sed 's/NAME=//g' | sed 's/\"//g' | tr " " '_')
+    if [ "$NAME" == "Fedora" ]; then
+    	NAME="Fedora_release"
+    fi
+    DIST="$(echo $NAME)_$(echo $VERSION)"
   elif [ -f /etc/mandriva-release ] ; then
-    DIST="`cat /etc/mandriva-release | sed 's/\ for .*//' | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/mandriva-release | sed 's/\ for .*//' | sed 's/\ (.*)//')"
   elif [ -f /etc/redhat-release ] ; then
-    DIST="`cat /etc/redhat-release | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/redhat-release | sed 's/\ (.*)//')"
   elif [ -f /etc/SuSE-release ] ; then
-    DIST="`cat /etc/SuSE-release | tr '\n' ' ' | sed 's/\ VERSION.*//' | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/SuSE-release | tr '\n' ' ' | sed 's/\ VERSION.*//' | sed 's/\ (.*)//')"
   elif [ -f /etc/debian_version ] ; then
-    DIST="`echo Debian` `cat /etc/debian_version | sed 's/\//-/'`"
+    DIST="$(echo Debian) $(cat /etc/debian_version | sed 's/\//-/')"
   fi
 fi
 
diff -up qmc2-0.43/sdlmess/scripts/os-detect.sh.os-detect qmc2-0.43/sdlmess/scripts/os-detect.sh
--- qmc2-0.43/sdlmess/scripts/os-detect.sh.os-detect	2014-03-13 18:04:23.612684676 +0100
+++ qmc2-0.43/sdlmess/scripts/os-detect.sh	2014-04-26 19:28:22.427919690 +0200
@@ -7,15 +7,20 @@ DIST="`uname -r`"
 
 if [ "${OS}" = "Linux" ] ; then
   if [ -f /etc/os-release ] ; then
-    DIST="`cat /etc/os-release | tr '\n' ' ' | sed 's/\ VERSION_ID.*//' | sed 's/\ (.*)//' | sed 's/NAME=//g' | sed 's/VERSION=//g' | sed 's/\"//g'`"
+    VERSION=$(cat /etc/os-release | egrep "\bVERSION_ID\b" | sed 's/VERSION_ID=//g' | sed 's/\"//g')
+    NAME=$(cat /etc/os-release | egrep "\bNAME\b" | sed 's/NAME=//g' | sed 's/\"//g' | tr " " '_')
+    if [ "$NAME" == "Fedora" ]; then
+    	NAME="Fedora_release"
+    fi
+    DIST="$(echo $NAME)_$(echo $VERSION)"
   elif [ -f /etc/mandriva-release ] ; then
-    DIST="`cat /etc/mandriva-release | sed 's/\ for .*//' | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/mandriva-release | sed 's/\ for .*//' | sed 's/\ (.*)//')"
   elif [ -f /etc/redhat-release ] ; then
-    DIST="`cat /etc/redhat-release | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/redhat-release | sed 's/\ (.*)//')"
   elif [ -f /etc/SuSE-release ] ; then
-    DIST="`cat /etc/SuSE-release | tr '\n' ' ' | sed 's/\ VERSION.*//' | sed 's/\ (.*)//'`"
+    DIST="$(cat /etc/SuSE-release | tr '\n' ' ' | sed 's/\ VERSION.*//' | sed 's/\ (.*)//')"
   elif [ -f /etc/debian_version ] ; then
-    DIST="`echo Debian` `cat /etc/debian_version | sed 's/\//-/'`"
+    DIST="$(echo Debian) $(cat /etc/debian_version | sed 's/\//-/')"
   fi
 fi
 
