diff -up qmc2-0.2/sdlmame/inst/qmc2.ini.template.ini qmc2-0.2/sdlmame/inst/qmc2.ini.template
--- qmc2-0.2/sdlmame/inst/qmc2.ini.template.ini	2009-07-22 23:25:59.000000000 +0200
+++ qmc2-0.2/sdlmame/inst/qmc2.ini.template	2010-05-16 12:25:51.000000000 +0200
@@ -8,6 +8,7 @@ qmc2-sdlmame\FilesAndDirectories\Control
 qmc2-sdlmame\FilesAndDirectories\MarqueeDirectory=DATADIR/qmc2/mrq/
 qmc2-sdlmame\FilesAndDirectories\TitleDirectory=DATADIR/qmc2/ttl/
 qmc2-sdlmame\FilesAndDirectories\PCBDirectory=DATADIR/qmc2/pcb/
+qmc2-sdlmame\Gamelist\UseCatverIni=true
 qmc2-sdlmess\FilesAndDirectories\DataDirectory=DATADIR/qmc2/
 qmc2-sdlmess\FilesAndDirectories\PreviewDirectory=DATADIR/qmc2/prv/
 qmc2-sdlmess\FilesAndDirectories\FlyerDirectory=DATADIR/qmc2/fly/
@@ -19,7 +20,43 @@ qmc2-sdlmess\FilesAndDirectories\TitleDi
 qmc2-sdlmess\FilesAndDirectories\PCBDirectory=DATADIR/qmc2/pcb/
 
 [MAME]
+Configuration\Global\rompath="DATADIR/mame/roms;DATADIR/mame/chds"
+Configuration\Global\samplepath=DATADIR/mame/samples
+Configuration\Global\artpath="DATADIR/mame/artwork;DATADIR/mame/effects"
+Configuration\Global\cheat_file=DATADIR/mame/cheat.dat
+Configuration\Global\ctrlrpath=DATADIR/mame/ctrlr
+Configuration\Global\cfg_directory=$HOME/.mame/cfg
+Configuration\Global\comment_directory=$HOME/.mame/comments
+Configuration\Global\diff_directory=$HOME/.mame/diff
+Configuration\Global\inipath="$HOME/.mame/ini;/etc/mame"
+Configuration\Global\input_directory=$HOME/.mame/inp
+Configuration\Global\memcard_directory=$HOME/.mame/memcard
+Configuration\Global\nvram_directory=$HOME/.mame/nvram
+Configuration\Global\snapshot_directory=$HOME/.mame/snap
+Configuration\Global\state_directory=$HOME/.mame/sta
+Configuration\Global\video=opengl
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMAME/template.xml
+FilesAndDirectories\ExecutableFile=/usr/bin/mame
+FilesAndDirectories\EmuInfoDB=DATADIR/mame/mameinfo.dat
+FilesAndDirectories\GameInfoDB=DATADIR/mame/history.dat
+FilesAndDirectories\CatverIni=DATADIR/mame/Catver.ini
 
 [MESS]
+Configuration\Global\rompath="DATADIR/mess/roms;DATADIR/mess/chds"
+Configuration\Global\samplepath=DATADIR/mess/samples
+Configuration\Global\artpath="DATADIR/mess/artwork;DATADIR/mess/effects"
+Configuration\Global\cheat_file=DATADIR/mess/cheat.dat
+Configuration\Global\ctrlrpath=DATADIR/mess/ctrlr
+Configuration\Global\cfg_directory=$HOME/.mess/cfg
+Configuration\Global\comment_directory=$HOME/.mess/comments
+Configuration\Global\diff_directory=$HOME/.mess/diff
+Configuration\Global\inipath="$HOME/.mess/ini;/etc/mess"
+Configuration\Global\input_directory=$HOME/.mess/inp
+Configuration\Global\memcard_directory=$HOME/.mess/memcard
+Configuration\Global\nvram_directory=$HOME/.mess/nvram
+Configuration\Global\snapshot_directory=$HOME/.mess/snap
+Configuration\Global\state_directory=$HOME/.mess/sta
+Configuration\Global\video=opengl
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMESS/template.xml
+FilesAndDirectories\ExecutableFile=/usr/bin/mess
+FilesAndDirectories\GameInfoDB=DATADIR/mess/sysinfo.dat
diff -up qmc2-0.2/sdlmess/inst/qmc2.ini.template.ini qmc2-0.2/sdlmess/inst/qmc2.ini.template
--- qmc2-0.2/sdlmess/inst/qmc2.ini.template.ini	2009-07-22 23:25:59.000000000 +0200
+++ qmc2-0.2/sdlmess/inst/qmc2.ini.template	2010-05-16 12:25:51.000000000 +0200
@@ -8,6 +8,7 @@ qmc2-sdlmame\FilesAndDirectories\Control
 qmc2-sdlmame\FilesAndDirectories\MarqueeDirectory=DATADIR/qmc2/mrq/
 qmc2-sdlmame\FilesAndDirectories\TitleDirectory=DATADIR/qmc2/ttl/
 qmc2-sdlmame\FilesAndDirectories\PCBDirectory=DATADIR/qmc2/pcb/
+qmc2-sdlmame\Gamelist\UseCatverIni=true
 qmc2-sdlmess\FilesAndDirectories\DataDirectory=DATADIR/qmc2/
 qmc2-sdlmess\FilesAndDirectories\PreviewDirectory=DATADIR/qmc2/prv/
 qmc2-sdlmess\FilesAndDirectories\FlyerDirectory=DATADIR/qmc2/fly/
@@ -19,7 +20,43 @@ qmc2-sdlmess\FilesAndDirectories\TitleDi
 qmc2-sdlmess\FilesAndDirectories\PCBDirectory=DATADIR/qmc2/pcb/
 
 [MAME]
+Configuration\Global\rompath="DATADIR/mame/roms;DATADIR/mame/chds"
+Configuration\Global\samplepath=DATADIR/mame/samples
+Configuration\Global\artpath="DATADIR/mame/artwork;DATADIR/mame/effects"
+Configuration\Global\cheat_file=DATADIR/mame/cheat.dat
+Configuration\Global\ctrlrpath=DATADIR/mame/ctrlr
+Configuration\Global\cfg_directory=$HOME/.mame/cfg
+Configuration\Global\comment_directory=$HOME/.mame/comments
+Configuration\Global\diff_directory=$HOME/.mame/diff
+Configuration\Global\inipath="$HOME/.mame/ini;/etc/mame"
+Configuration\Global\input_directory=$HOME/.mame/inp
+Configuration\Global\memcard_directory=$HOME/.mame/memcard
+Configuration\Global\nvram_directory=$HOME/.mame/nvram
+Configuration\Global\snapshot_directory=$HOME/.mame/snap
+Configuration\Global\state_directory=$HOME/.mame/sta
+Configuration\Global\video=opengl
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMAME/template.xml
+FilesAndDirectories\ExecutableFile=/usr/bin/mame
+FilesAndDirectories\EmuInfoDB=DATADIR/mame/mameinfo.dat
+FilesAndDirectories\GameInfoDB=DATADIR/mame/history.dat
+FilesAndDirectories\CatverIni=DATADIR/mame/Catver.ini
 
 [MESS]
+Configuration\Global\rompath="DATADIR/mess/roms;DATADIR/mess/chds"
+Configuration\Global\samplepath=DATADIR/mess/samples
+Configuration\Global\artpath="DATADIR/mess/artwork;DATADIR/mess/effects"
+Configuration\Global\cheat_file=DATADIR/mess/cheat.dat
+Configuration\Global\ctrlrpath=DATADIR/mess/ctrlr
+Configuration\Global\cfg_directory=$HOME/.mess/cfg
+Configuration\Global\comment_directory=$HOME/.mess/comments
+Configuration\Global\diff_directory=$HOME/.mess/diff
+Configuration\Global\inipath="$HOME/.mess/ini;/etc/mess"
+Configuration\Global\input_directory=$HOME/.mess/inp
+Configuration\Global\memcard_directory=$HOME/.mess/memcard
+Configuration\Global\nvram_directory=$HOME/.mess/nvram
+Configuration\Global\snapshot_directory=$HOME/.mess/snap
+Configuration\Global\state_directory=$HOME/.mess/sta
+Configuration\Global\video=opengl
 FilesAndDirectories\OptionsTemplateFile=DATADIR/qmc2/opt/SDLMESS/template.xml
+FilesAndDirectories\ExecutableFile=/usr/bin/mess
+FilesAndDirectories\GameInfoDB=DATADIR/mess/sysinfo.dat
